### PR TITLE
Update translations

### LINKED ISSUE #17: Language switcher and Translation
- [ ]  Language switcher support
- [ ] Translation of the web to vietnamese as per the documentation in this folder [here](https://drive.google.com/drive/folders/1IF6HTEGkxLHw6uK6M14TME3ILEVvCTu1?usp=sharing) [webapp]

### PR DESCRIPTION
**Pull request by peterMuriuki**
update vi translation files.
Re-audit for missing translations
Refactor module card pages so that language change does not break styles.
 close #17 
 
To Transfer to issue:
 - [ ] create an issue to update new strings.
 - [ ] plurals are not quite working correctly

### CODE REVIEW

**Review by moshthepitt — APPROVED** [review]

**Review by KipSigei — APPROVED** [review]

File: src/containers/Compartments/index.tsx [hunk 1]
```diff
@@ -356,10 +358,14 @@ export const Compartments = ({ module }: Props) => {
                     getNumberOfSmsWithRisk(LOW, woman, RISK_LEVEL) +
                     getNumberOfSmsWithRisk(HIGH, woman, RISK_LEVEL) +
                     getNumberOfSmsWithRisk(RISK, woman, RISK_LEVEL),
-                title: `${woman.length} Total Mother${woman.length > 1 ? 's' : ''} in PNC`,
+                title: format(
+                    t('total_mother_in_pnc.%d %s'),
+                    woman.length,
+                    t('Total mother in PNC', { count: woman.length }),
```
Comment (machariamuguku) [inline src/containers/Compartments/index.tsx:364]: this won't pluralise mother when count is > 1

File: src/containers/HierarchichalDataTable/index.tsx [hunk 2]
```diff
@@ -541,23 +538,25 @@ const getTotals = (dataToShow: LocationWithData[], module: moduleType) => {
 const unavailableChildren = (headerTitle: string[], currentLevel: 0 | 1 | 2 | 3, t: TFunction) => {
     // translate level number to level name
     const levelToName = (levelNo: 0 | 1 | 2 | 3, t: TFunction, plural?: boolean) => {
-        let levelName: 'provinces' | 'province' | 'districts' | 'district' | 'communes' | 'commune' | 'villages';
+        let levelName: string;
```
Comment (machariamuguku) [inline src/containers/HierarchichalDataTable/index.tsx:541]: why?

**Review by machariamuguku — COMMENTED** [review]
@p-netm Post review but

File: src/containers/HierarchichalDataTable/index.tsx (hunk 2 above)
Comment (peterMuriuki) [reply src/containers/HierarchichalDataTable/index.tsx:541]: levelName will hold the translated values of these values, too.

File: src/containers/Compartments/index.tsx (hunk 1 above)
Comment (peterMuriuki) [reply src/containers/Compartments/index.tsx:364]: tracking that from here: https://github.com/opensrp/miecd/issues/50

**Pull request merged**